flink-streaming-java/src/main/java/org/apache/flink/streaming/runtime/operators/windowing/WindowOperator.java

```diff
@@ -132,6 +133,13 @@
 	 */
 	protected final OutputTag<IN> lateDataOutputTag;
 
+	/**
+	* Metrics about the lost data due to arrive late.
+	* */
+	protected final String loseData = "lost_data";
+
+	protected Counter lostDataCount;
```

Comment:
Should this be registered on Flink metric system?

Reply:
Do you mean that i should add this in OperatorIOMetricGroup?

Reply:
but i think the OperatorIOMetricGroup is all about IO Metrics about Operator, is it suitable?

Reply:
the counter is registered already in the open method upon initialization, i think we're good on that front.

However, i would suggest to change this line to `protected final Counter lostDataCount = new SimpleCounter()`, and the line in open() to `metrics.counter(lostData, this.lostDataCount);`

Reply:
@zentol  i have adjust my code according the comment, and add the doc about this metrics, please review again, thanks

Reply:
@zentol if i use `protected final Counter lostDataCount = new SimpleCounter()` then i run into `Caused by: java.io.NotSerializableException: org.apache.flink.metrics.SimpleCounter` because `SimpleCounter` is not Serializable, so i think i should use the old way that `	this.lostDataCount = metrics.counter(LATE_ELEMENTS_METRIC_NAME);` in open method